src/OrchardCore.Modules/OrchardCore.Lucene/Controllers/SearchController.cs

```diff
@@ -137,8 +136,24 @@ public async Task<IActionResult> Search(SearchIndexViewModel viewModel, PagerSli
                 end = Convert.ToInt32(pagerParameters.After) + pager.PageSize + 1;
             }
 
-            var contentItemIds = (await _searchQueryService.ExecuteQueryAsync(query, searchSettings.SearchIndex, start, end))
-                .ToList();
+            var terms = viewModel.Terms;
+            if (!searchSettings.AllowLuceneQueriesInSearch)
+            {
+                terms = QueryParser.Escape(terms);
+            }
+
+            IList<string> contentItemIds;
```

Comment:
Why are we using List here not `IEnumerable and Enumerable.Empty<string>()`?

I know it's not your change @thomasebsmith but is there something about lucene that requires us to call `.ToList()` (@Skrypt ?) 

I think for errors, we should probably log them at least, and I _think_ it would be ok to add a generic `Incorrect query syntax` error to the `ModelState` dictionary for the `Terms` property

Reply:
I don't like this code either. We should never use a Try / Catch to silently avoid a potential error without logging it or returning an error message to the user so that he understands that his query failed to return some result(s). 

@deanmarcussen The ToList() is necessary here if we look later we are doing contentItemIds.IndexOf() with it which would require an Extension of the IEnumerable so that it works which would not make any sense since ...

Reply:
@deanmarcussen Actually, calling `.ToList()` was part of one of my previous changes (#6798). It is needed for `contentItemIds.IndexOf`, which is called later in the controller.